tests/ImageSharp.Tests/Formats/Png/PngDecoderTests.cs

```diff
@@ -20,79 +20,105 @@ public class PngDecoderTests
     {
         private const PixelTypes PixelTypes = Tests.PixelTypes.Rgba32 | Tests.PixelTypes.RgbaVector | Tests.PixelTypes.Argb32;
 
+        // TODO: Cannot use exact comparer since System.Drawing doesn't preserve more than 32bits.
+        private static readonly ImageComparer ValidatorComparer = ImageComparer.TolerantPercentage(0.1302F, 2134);
```

Comment:
Even if S.D decoder is inaccurate, and the output seems visually perfect, this high level of tolerance seems worrying for me. It might be a sign that something is wrong somewhere (maybe the comparison logic). What is the percentage difference if you lower the threshold?